beaglev.md

```diff
@@ -0,0 +1,392 @@
+BeagleV
+=======
+
+BeagleV is a low-cost RISC-V 64-bit based platform. The beta revision of the
+board uses the Starfive JH7100 SoC.
+
+How to build
+============
+
+```
+$ SHELL=/bin/bash kas-container build ./meta-riscv/beaglev.yml
+```
+
+The `kas-container` script can be obtained from
+[kas repository](https://github.com/siemens/kas/blob/2.4/kas-container).
```

Comment:
Could we also add a section saying to just run MACHINE=beaglev-starlight-jh7100 bitbake core-image-minimal if OE is already set up?